sdk/cloudmachine/Azure.Projects/src/Ofx/MessagingServices.cs

```diff
@@ -0,0 +1,69 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.ClientModel.Primitives;
+using System.Runtime.CompilerServices;
+using System.Threading.Tasks;
+using Azure.Core;
+using Azure.Messaging.ServiceBus;
+
+namespace Azure.Projects;
+
+/// <summary>
+/// The messaging services for the project client.
+/// </summary>
+public readonly struct MessagingServices
+{
+    internal const string DEFAULT_SB_TOPIC = "cm_servicebus_default_topic";
```

Comment:
Do we want to rename these `cm_...` strings here like we did elsewhere?

Reply:
yes, I am slowly weeding them out. But because we don't have live tests these changes are so fragile, that I don't want to do this one now. It would require me to go throug the whole manual process of provisioning, testing, etc.